test/statsCases/exclude-with-loader/expected.txt

```diff
@@ -1,7 +1,7 @@
-Hash: 9a949e8727d0583cb1c4
+Hash: 14e131b4e5c91ed5ce4a
 Time: Xms
     Asset     Size  Chunks             Chunk Names
-bundle.js  2.74 kB       0  [emitted]  main
-   [0] (webpack)/test/statsCases/exclude-with-loader/index.js 46 bytes {0} [built]
+bundle.js  2.92 kB       0  [emitted]  main
```

Comment:
Similar to the modules `+ 2 hidden modules` it would be great to add a note for assets too `+ 1 hidden asset`